### PR TITLE
 	3.3.0 release

### PR DESCRIPTION
**Pull request by JeroenDeDauw**

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
@kghbln @mwjames all fine by you?

File: INSTALL.md [hunk 1]
```diff
@@ -14,12 +14,18 @@ These are the installation and configuration instructions for the [SemanticMaps]
...
-		<th><a href="RELEASE-NOTES.md">Semantic Maps 3.2.0</a></th>
+		<th><a href="RELEASE-NOTES.md">Semantic Maps 3.3.0</a></th>
 		<td>Stable release</td>
-		<td>2015-07-02</td>
+		<td>2016-04-01</td>
 		<td><a href="https://github.com/JeroenDeDauw/SemanticMaps/tree/master">master</a></td>
 	</tr>
 	<tr>
+		<th><a href="RELEASE-NOTES.md">Semantic Maps 3.2.0</a></th>
+		<td>Legacy release</td>
```
Comment (kghbln) [inline INSTALL.md]: Legacy should all be Obsolete since we do not really support these versions any longer.

**Comment by kghbln** [comment]
> @kghbln all fine by you?

Just a comment. No hold-up.

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 These are the release notes for the Semantic Maps extension.
 
+## Semantic Maps 3.3
+
+Released on April 1st, 2016.
+
+* Added `userparam` support for the map result formats (by James Hong Kong)
+* Made Google Maps initialization more robust (by Karlpietsch)
+* Added missing system messages (by Karsten Hoffmeyer)
+
```
Comment (kghbln) [inline RELEASE-NOTES.md]: Ah, on thing is missing:

*  Migrated wfMsg to wfMessage (by Karsten Hoffmeyer)

see https://github.com/SemanticMediaWiki/SemanticMaps/commit/2ceef13709c7ee4cacbd010c428bff0fb75469b3

File: INSTALL.md [hunk 3]
```diff
@@ -120,7 +120,9 @@ versions prior to 1.22 can use Composer via the
 [Extension Installer](https://github.com/JeroenDeDauw/ExtensionInstaller/blob/master/README.md)
 extension.
 
-Note that the [Semantic MediaWiki](https://semantic-mediawiki.org/wiki/Help:Installation) extension as well as the  [Maps](https://github.com/JeroenDeDauw/Maps/blob/master/docs/INSTALL.md) extension need to be installed first for this extension to work.
+Note that the [Semantic MediaWiki](https://semantic-mediawiki.org/wiki/Help:Installation) extension as
```
Comment (kghbln) [inline INSTALL.md]: and a ``<www.sem>``

**Comment by kghbln** [comment]
@JeroenDeDauw I will try to hijack you pull since a MW 1.26+ fix needs to be added.

**Comment by kghbln** [comment]
@JeroenDeDauw I failed to do so. So do not tag. I will still move in a pull for 3.3

File: RELEASE-NOTES.md (hunk 2 above)
Comment (JeroenDeDauw) [inline RELEASE-NOTES.md]: I don't think this is really of interest to users. It impacts compat at some point, though that is already communicated differently

**Comment by JeroenDeDauw** [comment]
Please create a distinct PR if you have a fix for something

File: RELEASE-NOTES.md (hunk 2 above)
Comment (kghbln) [inline RELEASE-NOTES.md]: :P no worries. this was just me trying to dominate the world.

**Comment by kghbln** [comment]
> Please create a distinct PR if you have a fix for something

Git told me that I was moving on fast forward. Well I indented to move on forward fast so there was a misconception by Git. Ha, ha, ha. Spaß beiseite: I have to fiddle with the some file you changed in this pull so I suspect merge conflicts. Anyway I will do the distinct thing now.

**Comment by kghbln** [comment]
Done with https://github.com/SemanticMediaWiki/SemanticMaps/pull/58

**Comment by JeroenDeDauw** [comment]
We're all good now then right? (If so, I'll make the actual tag)

**Comment by kghbln** [comment]
Suppose :)

**Pull request merged**